examples/resources/gitlab_topic/resource.tf

```
@@ -0,0 +1,4 @@
+resource "gitlab_topic" "functional-programming" {
```

Style - underscores in names.

```suggestion
resource "gitlab_topic" "functional_programming" {
```